Cannot find symbol in Heimdal

Compiling master (4b9f7e54799418bfe604f9977044c49e21b54198) fails with Heimdal in FreeBSD 11.1-RELEASE base:

```
Compiling krb5wrap.cpp
c++ -I. -I/usr/local/include  -I/usr/include -g -O2 -Wall -Wextra -Wno-write-strings -c krb5wrap.cpp -o krb5wrap.o
In file included from krb5wrap.cpp:31:
In file included from ./msktutil.h:350:
./krb5wrap.h:201:24: error: no member named 'key' in 'krb5_keytab_entry'
        return m_entry.key;
               ~~~~~~~ ^
1 error generated.
*** Error code 1

Stop.
```

While the defition in Heimdal is: https://github.com/heimdal/heimdal/blob/master/lib/krb5/krb5.h#L566-L573

This definitvely requires an `#ifdef`:
```
diff --git a/krb5wrap.h b/krb5wrap.h
index 5bda2ea..fbcc4a8 100644
--- a/krb5wrap.h
+++ b/krb5wrap.h
@@ -198,7 +198,11 @@ public:
     krb5_timestamp timestamp() const { return m_entry.timestamp; }

     krb5_keyblock key() const {
+#ifdef HEIMDAL
+        return m_entry.keyblock;
+#else
         return m_entry.key;
+#endif
     }
 };
```
Since my C++ knowledge is non existent, I do not know whether this requires a static cast like the block above.

--- a/krb5wrap.h
+++ b/krb5wrap.h
@@ -198,7 +198,11 @@ public:
     krb5_timestamp timestamp() const { return m_entry.timestamp; }
 
     krb5_keyblock key() const {
-        return m_entry.key;
+#ifdef HEIMDAL
+       return m_entry.keyblock;
+#else
+       return m_entry.key;
+#endif
     }
 };
 

